app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts
+    @posts = Post.includes(:comments).all
```

Comment:
- [ ] Good work on fixing the N+1 problem however you should select only the user posts, not all the posts

```suggestion
    @posts = @user.posts.includes(:comments)
```